test-infra/camel-test-infra-zookeeper/src/test/java/org/apache/camel/test/infra/zookeeper/services/ZooKeeperContainer.java

```diff
@@ -23,7 +23,12 @@
 import org.testcontainers.containers.wait.strategy.Wait;
 
 public class ZooKeeperContainer extends GenericContainer {
-    public static final String CONTAINER_IMAGE = "zookeeper:3.5";
+    if (arch == ppc64le) {
+        public static final String CONTAINER_IMAGE = "icr.io/ppc64le-oss/zookeeper-ppc64le:v3.8.0-bv"
+    } else {
+        public static final String CONTAINER_IMAGE = "zookeeper:3.5";
+    }
+    
```

Comment:
I'm not sure that it can compile, maybe you meant: 

`public static final String CONTAINER_IMAGE = arch == ppc64le ? "icr.io/ppc64le-oss/zookeeper-ppc64le:v3.8.0-bv" : "zookeeper:3.5";`

or 
```
   public static final String CONTAINER_IMAGE;
    if (arch == ppc64le) {
        CONTAINER_IMAGE = "icr.io/ppc64le-oss/zookeeper-ppc64le:v3.8.0-bv"
    } else {
        CONTAINER_IMAGE = "zookeeper:3.5";
    }
```

Reply:
@essobedo locally all the above 3 changes are working.
`public static final String CONTAINER_IMAGE = arch == ppc64le ? "icr.io/ppc64le-oss/zookeeper-ppc64le:v3.8.0-bv" : "zookeeper:3.5";`
Let me use above way to use the zoo-keeper image, as it will optimize the code. Thanks for your review.